src/client/src/models/Scene.js

```diff
@@ -41,41 +41,66 @@ const Scene = {
     return JSON.parse(localStorage.getItem('scene-list'))
   },
 
+  findSceneIndex: function(sceneid) {
+    // get current list
+    const list = JSON.parse(localStorage.getItem('scene-list'))
```

Comment:
this is used a lot despite getScenes() available, not a big deal just a nit pick

Reply:
getScene return the JSON of Scenes and findSceneIndex search the JSON for the object index because the scene id isn't equal to index anymore

Reply:
line 46: performs a call to load in the scene list to a JavaScript array but there is a function getScenes declared on line 40 that is cleaner than having it littered through. Not necessary but just pointing it out